andela/project-arakari
# #137011075 Include bundling task in gulp task runner

#### What does this PR do?
Adds bundling task in gulp task runner
fixes the problem with the coverage badge and coveralls progress
 #### Description of Task to be completed?
Include bundling task in gulp task runner
 #### How should this be manually tested?
After installing and running `npm install` check to see if there is a ./public/bundle folder
for the coverage check this branches coverage badge it reads 44%
 #### Any background context you want to provide?
 #### What are the relevant pivotal tracker stories?
#137011075
